glean-core/src/util.rs

```diff
@@ -111,6 +111,64 @@ pub(crate) fn truncate_string_at_boundary_with_error<S: Into<String>>(
     }
 }
 
+// On i686 on Windows, the CPython interpreter sets the FPU precision control
+// flag to 53 bits of precision, rather than the 64 bit default. This causes
+// different floating point results than on other architectures. This context
+// manager makes it easy to set the precision to 64-bits around floating point
+// operations.
+//
+// See https://bugzilla.mozilla.org/show_bug.cgi?id=1623335 for additional context.
+#[cfg(all(target_arch = "x86", target_os = "windows"))]
```

Comment:
Should this be limited to `target_env = "mscv"` additionally?
See [docs](https://doc.rust-lang.org/reference/conditional-compilation.html#target_env).